src/engraving/libmscore/edit.cpp

```diff
@@ -453,7 +453,7 @@ Rest* Score::setRest(const Fraction& _tick, int track, const Fraction& _l, bool
             // compute list of durations which will fit l
             //
             std::vector<TDuration> dList;
-            if (tuplet || staff->isLocalTimeSignature(tick)) {
+            if (tuplet || staff->isLocalTimeSignature(tick) || f == Fraction(0, 1)) {
```

Comment:
was crashing without this when pasting, e.g. a whole note onto a multi-measure rest, the 'else' part of this test definitely doesn't work if `f` is 0 anyway